test/Constraints/rdar33142387.swift

```diff
@@ -0,0 +1,37 @@
+// RUN: %target-swift-frontend -emit-sil -verify %s | %FileCheck %s
```

Comment:
You can use `-emit-silgen` here rather than `-emit-sil`, which avoids running the mandatory passes. I believe that's what `%target-swift-emit-silgen` should do.